### PR TITLE
Support ctdir

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

File: lib/puppet/type/firewall.rb [hunk 1]
```diff
@@ -840,6 +840,10 @@ def should_to_s(value)
     end
   end
 
+  newproperty(:ctdir) do
+    newvalues(:REPLY, :ORIGINAL)
```
Comment (eimlav) [inline lib/puppet/type/firewall.rb]: Preferably there should be a brief `desc` here

Comment (raphink) [reply lib/puppet/type/firewall.rb]: I added a desc

**Comment by eimlav** [comment]
@raphink Thanks for updating the `desc`. Please see my previous comment for the other changes that are needed. Let me know if you have any questions about these.

**Comment by eimlav** [comment]
Hi @raphink is this PR still something you are looking to continue working on or are you happy with it being closed? Thanks.

**Comment by raphink** [comment]
Sorry for the delay @eimlav. I intend to get back to it indeed.

**Comment by lionce** [comment]
close and reopen to rerun checks :)

**Comment by lionce** [comment]
Hello @raphink ,

Please let us know if you intend to continue the work for this PR. It's looking good to me. It just needs acceptance tests for your changes. Please let us know if you need any help with that! Thanks

Cheers!

**Comment by lionce** [comment]
Hello @raphink ,

Thank you for this contribution. It looks good!

Cheers!

**Comment by DavidS** [comment]
This is superseded by #872 which has a more complete support for the ct options.

**Pull request closed without merging**